.evergreen/config/functions.yml

```diff
@@ -274,12 +274,15 @@ functions:
           export KMS_TLS_CA_FILE="${client_side_encryption_kms_tls_ca_file}"
           export KMS_TLS_CERTIFICATE_KEY_FILE="${client_side_encryption_kms_tls_certificate_key_file}"
           export MONGODB_IS_SERVERLESS=on
-          export MONGODB_USERNAME=${SERVERLESS_ATLAS_USER}
-          export MONGODB_PASSWORD=${SERVERLESS_ATLAS_PASSWORD}
+          export MONGODB_URI="${SERVERLESS_URI}"
           export PATH="${PHP_PATH}/bin:$PATH"
 
+          . ${DRIVERS_TOOLS}/.evergreen/serverless/secrets-export.sh
+
+          export MONGODB_USERNAME=$SERVERLESS_ATLAS_USER
+          export MONGODB_PASSWORD=$SERVERLESS_ATLAS_PASSWORD
```

Comment:
FYI, it took me too long to remember that `$FOO` refers to an environment variable named `FOO`, whereas `${FOO}` refers to an evergreen expansion named `FOO` 🤦‍♂️